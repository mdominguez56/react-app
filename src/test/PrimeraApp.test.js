import React from "react";
import "@testing-library/jest-dom";
// import { render } from "@testing-library/react";
import PrimeraApp from "../PrimeraApp";
import { shallow } from "enzyme";

describe("Prueba en PrimeraApp component", () => {
  // test('Debe mostrar el mensaje, "Hola soy Goku"', () => {
  //  const saludo = "Hola soy Goku";
  //  const { getByText } = render(<PrimeraApp saludo={saludo} />);
  //  expect(getByText(saludo)).toBeInTheDocument();
  //});

  test("Debe demostrar el componente correctamente", () => {
    const saludo = "Hola soy Goku";
    const wrapper = shallow(<PrimeraApp saludo={saludo} />);

    expect(wrapper).toMatchSnapshot();
  });
});
