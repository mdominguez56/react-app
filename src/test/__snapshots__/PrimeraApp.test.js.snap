// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Prueba en PrimeraApp component Debe demostrar el componente correctamente 1`] = `
<Fragment>
  <h1>
     
    Hola soy Goku
     
  </h1>
  <p>
     
    Soy un subtitulo
     
  </p>
</Fragment>
`;
